### PR TITLE
Feature/minor doc fixes

### PR DESCRIPTION
**Pull request by jennmedellin**
## Description
Fix minor margin, padding, and text issues in Covalent Docs examples.

### What's included?
- Do not use "Dismiss" per Material Design
> Reworked snackbars and alert in Design Patterns > Alerts
> Use "Ok" on alert and single word confirmation on other dialogs
- In Componets> Dynamic Forms > Dynamic Booleans example, effects cut off in the corner
> Add margin to td-dynamic-forms
- In Components > Paging > Paging with everything demo, prevent wrapping on resize
> Reduce default margin in paging component SCSS

#### Test Steps
- Verify items above

#### General Tests for Every PR

- [ ] `ng serve --aot` still works.
- [ ] `npm run lint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build` still works.

##### Design Patterns > Alerts examples
covalent_alert

##### Components > Dynamic Forms > Dyamic Boolean Elements example
covalent_boolean

##### Components > Paging
covalent_paging

### CODE REVIEW

File: src/app/components/components/paging/paging.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* Set margin on all elements in td-paging-bar to 5px to prevent wrapping in examples */
+.td-paging-bar * {
```
Comment (kyleledbetter) [inline src/app/components/components/paging/paging.component.scss]: hmmm i'm concerned with using custom scss in docs, it will potentially mislead users

**Comment by aciccarello** [comment]
BTW, this closes #844. Thanks @jennmedellin

**Comment by emoralesb05** [comment]
LGTM

File: src/app/components/components/paging/paging.component.scss (hunk 1 above)
Comment (jennmedellin) [reply src/app/components/components/paging/paging.component.scss]: Adjusted to breakpoints and link count instead of padding/margin.

**Pull request merged**